### ISSUE TITLE
Following install and run dev error with node v0.12 plus node v4.x request

### ISSUE THREAD

**Issue by AddictArts** [salesforce-ux/design-system#54]
Please consider adding engine to the package.json, so people like me don't try node v4.x. Thanks

I used node v0.12, npm i, then npm run dev and ...
```
Error: Option safe was removed. Use parser: require("postcss-safe-parser")
    at parse (/Users/admin/box/web/design-system/node_modules/postcss/lib/parse.js:18:15)
    at new LazyResult (/Users/admin/box/web/design-system/node_modules/postcss/lib/lazy-result.js:57:24)
    at Processor.process (/Users/admin/box/web/design-system/node_modules/postcss/lib/processor.js:36:16)
    at collectSassVariables (/Users/admin/box/web/design-system/node_modules/@salesforce-ux/aura-css-utils/dist/util.js:108:65)
    at Array.map (native)
    at collectSassVariablesFromFiles (/Users/admin/box/web/design-system/node_modules/@salesforce-ux/aura-css-utils/dist/util.js:129:6)
    at Object.<anonymous> (/Users/admin/box/web/design-system/node_modules/@salesforce-ux/design-system-utils/dist/index.js:75:25)
    at /Users/admin/box/web/design-system/node_modules/stampit/dist/stampit.js:185:31
    at arrayEach (/Users/admin/box/web/design-system/node_modules/lodash/internal/arrayEach.js:15:9)
    at /Users/admin/box/web/design-system/node_modules/lodash/internal/createForEach.js:15:9
```

It would be great if this worked right after clone. Consider something like for the engine requirement:
```
    "predev": "node -e 'if(/^v(2|3|4)/.test(process.version)===true){console.log(\"node engine error, looking for node v0.10 <= v0.12\");process.exit(1);}'",

```

**Comment by aputinski** [comment]
@AddictArts Sorry about that — I'll take a look. There isn't *supposed* to be a requirement like that, so it looks like I need to fix one of the dependencies. I'll try to take a look at that tomorrow.

**Comment by aputinski** [comment]
@AddictArts This should now be fixed in `0.10.0`. I was able to run locally using node `4.1.1`, but let me know if you have any issues.

**Comment by AddictArts** [comment]
Wow, thank you. I tested with node v4.1.2 and npm 3.3.6, fixed
